### ISSUE TITLE
Installed JDK now use Eclipse Adoptium as target folder

### ISSUE THREAD

**Issue by himinhee** [microsoft/vscode-java-installer#256]
:warning:Please attach **%TEMP%\vscode-java-installer\log.txt** and **%TEMP%\vscode-java-installer\AdoptOpenJDK-MSI.log** for more details.

version: 0.3.16
env:
```
JAVA_HOME: undefined
PATH:
C:\Program Files (x86)\Google\Chrome\Application
C:\WINDOWS\system32
C:\WINDOWS
C:\WINDOWS\System32\Wbem
C:\WINDOWS\System32\WindowsPowerShell\v1.0\
C:\WINDOWS\System32\OpenSSH\
C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL
C:\Program Files\Intel\Intel(R) Management Engine Components\DAL
C:\Program Files\nodejs\
C:\Program Files\Git\cmd
C:\Program Files\PuTTY\
C:\Program Files\Docker\Docker\resources\bin
C:\ProgramData\DockerDesktop\version-bin
C:\Program Files\MySQL\MySQL Shell 8.0\bin\
C:\Users\mostd\AppData\Local\Microsoft\WindowsApps
C:\Program Files\Bandizip\
C:\Users\mostd\AppData\Roaming\npm
C:\Users\mostd\AppData\Local\Programs\Microsoft VS Code\bin

JDKs:

```
log:
```
[오전 12:08:58] AdoptOpenJDK-11-x64
[오전 12:09:42] Download JDK ... [Completed]
[오전 12:09:42] Install JDK
[오전 12:09:42] Installing AdoptOpenJDK-11-x64...
[오전 12:10:27] Install JDK ... [Completed]
[오전 12:10:27] Configure Java Settings
[오전 12:10:28] Error occurred
[오전 12:10:28] Error: Java Home not found.

```

**Comment by Eskibear** [comment]
Adoptium JDK changed its target location. Corresponding fix will be shipped in 0.3.17.

**Comment by Eskibear** [comment]
shipped in v0.3.17
